components/common/src/command/package/install.rs

```diff
@@ -191,16 +197,19 @@ where
         return Err(Error::RootRequired);
     }
 
-    let cache_key_path = cache_key_path(Some(fs_root_path.as_ref()));
-    debug!("install cache_key_path: {}", cache_key_path.display());
+
+    // TODO (CM): rename fs::cache_key_path so the naming is
```

Comment:
Note that if we change one we'll want to change them all--they are all currently self consistent. But perhaps awkward.

Reply:
Agreed... I guess this comment was more to explain why _this_ variable didn't seem to match the function.